server/controllers/addPrisoner/addPrisoner.ts

```diff
@@ -64,6 +64,8 @@ export default class AddPrisoner extends FormInitialStep {
 
       const newPrisonerData: PrisonerInvolvement = {
         prisonerNumber: res.locals.prisoner.prisonerNumber as string,
+        firstName: 'Not',
+        lastName: 'Known',
```

Comment:
i'd say this doesn't matter since the whole add-a-prisoner process has not been designed so it's not worth the effort wiring this up